backend/app/views/spree/admin/shared/_sidebar_left.html.erb

```diff
@@ -0,0 +1,5 @@
+<div class='sidebar'>
```

Comment:
If there's only one main sidebar I'd prefer these to all just be `sidebar-#{thing}`.